components/Button/Button.tsx

```diff
@@ -110,6 +110,25 @@ const Button = styled.button<ButtonProps>`
 				}
 			}
 		`}
+
+		${(props) =>
+		props.variant === 'quaternary' &&
+		css`
+			background: ${(props) => props.theme.colors.black};
```

Comment:
is this supposed to just be black and white no matter which theme you are on?

Reply:
It's a button createed by @0xlinus and it's only used on the header, so I think it's okay.